python/tvm/relay/frontend/tensorflow.py

```diff
@@ -90,6 +90,12 @@ def _get_list_param(params, input_node):
 def _get_tuple_param(params, input_node):
     return tuple(_get_param(params, input_node))
 
+def _need_module_for_shape_inference(op):
+    return op in ['StridedSlice']
```

Comment:
could you explain the code block here? if stridedslice need to be handled specifically, not sure if some other ops like slicelike, arange also need it.

Reply:
Any operator that calls `_infer_shape` in the converter should be added here. `StridedSlice` calls `_infer_shape` so it needs to be added here.